samples/packaged/CopilotChat/WebApp/README.md

```diff
@@ -16,11 +12,7 @@
 4. Create an **[.env](.env)** file to this folder root with the following variables and fill in with your information, where `REACT_APP_GRAPH_CLIENT_ID=` is the GUID copied from the **Application (client) ID** from the Azure Portal:
         
         VITE_REACT_APP_CHAT_CLIENT_ID=
-        VITE_REACT_APP_FUNCTION_URI=http://localhost:7071
-        VITE_REACT_APP_AZURE_OPEN_AI_KEY=
-        VITE_REACT_APP_AZURE_OPEN_AI_COMPLETION_DEPLOYMENT=
-        VITE_REACT_APP_AZURE_OPEN_AI_EMBEDDINGS_DEPLOYMENT=
-        VITE_REACT_APP_AZURE_OPEN_AI_ENDPOINT=
+        VITE_REACT_APP_FUNCTION_URI=http://localhost:40443
```

Comment:
Should we add a comment to make sure this matches with what's in the SKWebApi appsettings?

Reply:
Addressed